server/routes/why-is-item-rmi.route.js

```diff
@@ -57,6 +57,8 @@ const handlers = {
 const _getContext = async request => {
   const whyRmi = await RedisService.get(request, RedisKeys.WHY_IS_ITEM_RMI)
 
+  console.log('whyRmi:', whyRmi)
```

Comment:
Think you may have left this console.log in here by accident

Reply:
Removed